informus/content/SPC_charts_hoverbox.md

```diff
@@ -61,15 +66,12 @@
 - Date:
 - Percentage:
 - Mean:
-- Incidents of delirium/Patients with delirium TBC:
-- Eligible patients:
```

Comment:
I have removed these labels from the CAM-ICU SPC tooltip as we agreed this in a separate conversation.